server/core/runtime/pre_workflow_hook_runner.go

```diff
@@ -71,7 +71,7 @@ func (wh DefaultPreWorkflowHookRunner) Run(ctx models.WorkflowHookCommandContext
 		}
 	}
 
-	wh.OutputHandler.SendWorkflowHook(ctx, string(out), false)
+	wh.OutputHandler.SendWorkflowHook(ctx, strings.ReplaceAll(string(out), "\n", "\r\n"), false)
```

Comment:
Please add tests for both pre and post changes

Reply:
Done.